### ISSUE TITLE
9.5 Staging 2226 - default chat tab does not actually show chat

### ISSUE THREAD

**Issue by MishkaFur** [StrangeLoopGames/EcoIssues#22986]
![image](https://user-images.githubusercontent.com/61889138/153778212-27f40491-3e95-4cd7-9fc2-9ac2228e44ee.png)

had to create a new tab for general to see my text

![image](https://user-images.githubusercontent.com/61889138/153778221-cf9aa700-aab3-4a68-96a4-5b84e659e3a2.png)

removing general from "Chat" then adding it again does nothing

**Comment by Daniel-Magulado** [comment]
Hi @MishkaFur 

Thank you for this report. This might be due to a bug that forces messages sent in the Chat tab to be on the top of the notification. Mind scrolling up and checking the top portion of the chat if the messages sent are there?
Thank you. Looking forward to your response.

**Comment by MishkaFur** [comment]
> Hi @MishkaFur
> 
> Thank you for this report. This might be due to a bug that forces messages sent in the Chat tab to be on the top of the notification. Mind scrolling up and checking the top portion of the chat if the messages sent are there? Thank you. Looking forward to your response.

They are not.

The message appears for a second or two then vanishes

**Comment by MonzUn** [comment]
If you scroll, the message is also gone 🤷

**Comment by Daniel-Magulado** [comment]
@MishkaFur 

Thank you for the response. Did it by chance that you encountered this bug on one of the live servers that you host? If so, is it possible for me to join? This will allow me to do some documentation of the bug for a report since we currently do not have a busy server to test it with. Upon testing, the one that I am encountering is the bug that forces the message sent by the player to be moved to the top of the chat.

If not, it's understandable. I will still make an internal report for it.

**Comment by MonzUn** [comment]
No, as there is no readily available staging client, I've only run 9.5 staging on my own test server.
But this triggers if you just run the server and join the client and send a message.

The message flickers quickly and then it's gone from the initial tab.
Of course, my test server also isn't unmodded as that's where I test my stuff ^^'

@MishkaFur Did you experience this before you added DiscordLink?
I create channels on startup so better to be sure that it's not interfering :)
